etcdctl/ctlv3/command/get_command.go

```diff
@@ -184,5 +192,21 @@ func getGetOp(args []string) (string, []clientv3.OpOption) {
 		opts = append(opts, clientv3.WithCountOnly())
 	}
 
+	if getMinCreateRev > 0 {
+		opts = append(opts, clientv3.WithMinCreateRev(getMinCreateRev))
+	}
+
+	if getMaxCreateRev > 0 {
+		opts = append(opts, clientv3.WithMaxCreateRev(getMaxCreateRev))
+	}
+
+	if getMinModRev > 0 {
+		opts = append(opts, clientv3.WithMinModRev(getMinModRev))
+	}
+
+	if getMaxModRev > 0 {
+		opts = append(opts, clientv3.WithMaxModRev(getMaxModRev))
+	}
```

Comment:
Should we check `getMinCreateRev <= getMinCreateRev` and `getMinModRev <= getMaxModRev`?

Reply:
Done.